BankexWallet/InterfaceElements/DesignElements/Onboarding/CustomNetworkViewController.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  CustomNetworkViewController.swift
+//  BankexWallet
+//
+//  Created by Георгий Фесенко on 01/06/2018.
+//  Copyright © 2018 Alexander Vlasov. All rights reserved.
+//
+
+import UIKit
+
+class CustomNetworkViewController: UIViewController {
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
```

Comment:
If you don't push, but instead replace the whole stack with `WalletCreationTypeController` you don't need to override prepareForSegue in tons of controllers

Reply:
Now, with only one view controller this is not a problem, I think